modules/administration/pages/tshoot-packages.adoc

```diff
@@ -19,22 +19,13 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 . Last step
 ////
 
-Packages can sometimes be locked or taskomatic can experience problems, which creates problems with metadata regeneration.
+Packages on a client can sometimes be locked, which creates problems with metadata regeneration.
 When this occurs, package updates will be available in the {webui}, but will not appear on the client, and attempts to update the client will fail.
-To correct this, determine if any processes are running, or if a crash could have occured.
 Check package locks and exclude lists to determine if packages are locked or excluded on the client.
 When you have located the problematic process, the metadata can be regenerated and synchronization occurs as expected.
```

Comment:
```suggestion
```

Rationale: once packages are unlocked on clients attempts to update it will not fail any more. Metadata regeneration has nothing to do with this.